src/project/manifest.rs

```diff
@@ -143,6 +144,24 @@ impl ProjectManifest {
                     self.build_dependencies.insert(IndexMap::new())
                 }
             }
+            SpecType::Pypi => panic!("Misusing function"),
+        }
+    }
+
+    /// Get the map of dependencies for a given spec type.
+    pub fn create_or_get_pypi_dependencies(
+        &mut self,
+        spec_type: SpecType,
```

Comment:
Can we get rid of this?

Reply:
Now using a `DependencyType` to split pypi and conda so removed it here